lib/click.ts

```diff
@@ -22,6 +22,12 @@ export function clickedObjectIDsAfterSearch(
   const { events, additionalParams } =
     extractAdditionalParams<InsightsSearchClickEvent>(params);
 
+  events.forEach(({ index, queryID, objectIDs }) =>
```

Comment:
Just checking that we want to store query IDs even if the user hasn't opted in the query ID inference? I guess this might make it more effective if they decide to enable it?

Reply:
There's no downside to storing it, and as you say once it's enabled it would instantly begin to work.